File: Resueltos_por_competidor/Miguel_Lopez/AntonandPolyhedrons.cpp
/*
 * Autor: Miguel Lopez
 * Problema: Anton and Polyhedrons
 * Juez online: Codeforces
 * Veredicto: Accepted
 * Url: https://codeforces.com/problemset/problem/785/A
 */

#include <bits/stdc++.h>
#include <cctype>
#include <cstdio>
#include <ios>
#include <ostream>
typedef long long int ll;
using namespace std;
void init_code() {
    ios_base::sync_with_stdio(false);
    cin.tie(nullptr);
    cout.tie(nullptr);
    #ifndef ONLINE_JUDGE
    freopen("input.txt", "r", stdin);
    freopen("output.txt", "w", stdout);
    #endif 
}
int main() {
    init_code();
    int n; cin >> n;
    map<string, int> faces = {{"Tetrahedron", 4},{"Cube", 6},{"Octahedron", 8},{"Dodecahedron", 12}, {"Icosahedron", 20}};
    ll total = 0;
    string s;
    while(n--) {
        cin >> s; total += faces[s];
    }

    cout << total << endl;
}

File: Resueltos_por_competidor/Jacobo_Segura/Anton_And_Polyhedrons.java
/* 
Codeforces 785A - Anton and Polyhedrons
https://codeforces.com/problemset/problem/785/A
*/
import java.util.Scanner;

public class Main {
	public static void main(String[] args) {
		Scanner sc = new Scanner(System.in);
		int n = sc.nextInt();
		int r = 0;
		for(int i = 0; i<n; i++) {
			String f = sc.next();
			if(f.equals("Icosahedron")) {
				r+=20;
			}else if(f.equals("Dodecahedron")) {
				r+=12;
			}else if(f.equals("Octahedron")) {
				r+=8;
			}else if(f.equals("Cube")) {
				r+=6;
			}else if(f.equals("Tetrahedron")) {
				r+=4;
			}
		}
		System.out.println(r);
	}
}
